changelog.d/6222.feature

```diff
@@ -0,0 +1 @@
+Synapse now supports the use of API tokens when interfacing with administration APIs. These tokens can be namespaced to allow access to particular endpoints, and actions on those endpoints.
```

Comment:
As per the company sync this week: we should avoid announcing new features until they are ready to be used. Could this just say "lay the groundwork for" or something and be a `.misc`?